cuda_core/pyproject.toml

```
@@ -48,6 +48,9 @@ dependencies = [
 [project.optional-dependencies]
 cu11 = ["cuda-bindings==11.8.*"]
 cu12 = ["cuda-bindings==12.*"]
+test = ["cython>=3.0", "distutils", "pytest>=6.2.4"]
```

distutils is now part of setuptools
```suggestion
test = ["cython>=3.0", "setuptools", "pytest>=6.2.4"]
```